packages/terra-search-field/src/SearchField.module.scss

```diff
@@ -47,7 +47,9 @@
       }
 
       &:focus {
-        border-color: var(--terra-search-field-button-focus-border-color, #26a2e5);
+        box-shadow: var(--terra-search-field-clear-focus-box-shadow);
+        outline: var(--terra-search-field-focus-keyboard-outline, 2px dashed #000);
```

Comment:
In this scenario, can't we create one descendant focus pseudo selector to cover all the use cases under the search field?  I don't think we should have to add a selector for each type of element.

Reply:
@cm9361 I was able to add the styles under the *.focus here https://github.com/cerner/terra-core/pull/3834/commits/4e9be890f0c2087ae529bf141f65a0ce25e667b7

Not sure if there's another approach you would apply here outside of the discussion had about having a library wide focus indicator style